lab-khalid/route/auth-router.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const
+  Router = require('express').Router,
+  jsonParser = require('body-parser').json(),
+  debug = require('debug')('CFgram:auth-router'),
+  authmiddleware = require('../lib/auth-middleware.js'),
+
+  User = require('../model/user.js'),
+
+  authRouter = module.exports = new Router();
```

Comment:
you don't need to instantiate a new router here. You can just call Router. 
`authRouter = module.exports = Router();`